model/src/main/kotlin/config/PackageCurationProviderConfiguration.kt

```diff
@@ -19,13 +19,16 @@
 
 package org.ossreviewtoolkit.model.config
 
+import com.fasterxml.jackson.annotation.JsonAlias
+
 import org.ossreviewtoolkit.utils.common.OrtPlugin
 
 data class PackageCurationProviderConfiguration(
     /**
      * The [name][OrtPlugin.type] of the package curation provider.
      */
-    val name: String,
+    @JsonAlias("name")
```

Comment:
The docs about should also be adjusted.

Reply:
Für Hoplite you need to add `@ConfigAlias`. I'm not sure if this is ever deserialized with Jackson so you might safely drop `@JsonAlias`.